adofai-roulette.py

```diff
@@ -2,74 +2,50 @@
 
 # setup song, difficulty, and creator lists
 
-tempfile = open("songs.txt", encoding="utf-8")
-songlistbad = tempfile.readlines()
-tempfile2 = open("difficulties.txt", encoding="utf-8")
-difficultylistbad = tempfile2.readlines()
-tempfile3 = open("creators.txt", encoding = "utf-8")
-creatorlistbad = tempfile3.readlines()
+def readLinesAndStrip(filename: str) -> list[str]:
+    with open(filename, encoding="utf-8") as file:
+        lines = file.readlines()
 
-songlist = []
-difficultylist = []
-creatorlist = []
+    return [line.strip() for line in lines]

```

Comment:
Note this is a list comprehension, which is a more advance python topic. In this case,
```python
result = [line.strip() for line in lines]
# is the same as
result = []
for line in lines:
    result.append(line.strip())
```
It just saves some space, and reduces unnecessary code.